docs/release/howto/udfs-in-pixeltable.ipynb

```diff
@@ -345,7 +345,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "\r",
+      "\r\n",
```

Comment:
When you save these files, Windows newlines get added all over the place. It's probably harmless, but we should really get you off Windows...

Reply:
1. Let's not waste time building an **Under Construction** page and on top of that who wants to click on a link that redirects them to an "Under construction" page that leads to no additional insights. 
2. Adding a <!-- TODO to our actual live documentation makes it look unfinished. It's distracting to the reader. We should just track internally what notebooks/documentation we want to expand in the future and not expose that WIP through commentary in the live doc.

Let's remove the sections mentioning future materials as we do not know when we might get to work on them and our plans change.

Regarding Windows, as you said, it's harmless and it's good to have one of us running Windows for testing.

Reply:
If you put a TODO in the format I proposed it will be an HTML comment that won't show up when the notebook cell is rendered.

Reply:
As discussed. Let's approve and merge it and add the TODO in HTML as we go where we feel like we need additions and/or as we build more tutorials.